src/main/java/com/levik/stealer/collect/repository/entity/PictureEntity.java

```diff
@@ -0,0 +1,37 @@
+package com.levik.stealer.collect.repository.entity;
+
+import com.levik.stealer.nasa.model.NasaImage;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Setter
+@Getter
+@Table(name = "pictures")
+@Entity
+public class PictureEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    @Column(name = "img_src", nullable = false)
+    private String imgSrc;
+
+    private LocalDateTime createdAt;
```

Comment:
Use OffsetDateTime instead of LocalDateTime. With OffsetDateTime, you can easily convert to other time zones or to UTC for comparisons and calculations. It simplifies operations like finding the difference between times or aligning events in a global context. OffsetDateTime helps accurately represent the point in time regardless of the user's location.